### ISSUE TITLE
SearchField should support readOnly

### ISSUE THREAD

**Issue by vignesh235** [maheshmnj/searchfield#75]
How to read for search field text?

**Comment by maheshmnj** [comment]
You can make use of TextEditingController similar to How you read text from a normal [TextField](https://api.flutter.dev/flutter/material/TextField-class.html)
 https://github.com/maheshmnj/searchfield/blob/fb7f9c91afb554e27c4270f2a3d839de5b14f94e/lib/src/searchfield.dart#L269
```dart
TexEditingController controller = new TextEditingController()
```

Attach a controller

```dart
SearchField(
 controller: controller
 ...
 ...
)
```

read text using 

```dart
final searchfieldText = controller.text;
```

And don't forget to dispose of the controller. Hope this answers your question, if it does then please feel free to close the issue.

**Comment by vignesh235** [comment]
how to make the read-only for the search text field.
In some case, I don't want to select the value , so I need that to make read only.

**Comment by maheshmnj** [comment]
Ah I see Thanks, we should definitely have a `readOnly`boolean, meanwhile you can try wrapping  [IgnorePointer](https://api.flutter.dev/flutter/widgets/IgnorePointer-class.html) around SearchField as a workaround.

Feel free to make a PR if you are interested. I will try to get this out by tomorrow.

**Comment by maheshmnj** [comment]
`readOnly` property is now available and released in [v0.7.6](https://pub.dev/packages/searchfield)
